### ISSUE TITLE
Ender Tanks recipes doesn't show on JEI

### ISSUE THREAD

**Issue by Fireztonez** [ShetiPhian/Localizations-and-Issues#208]
Hello,
I just install the EnderTanks mod i my modpack, but the recipes doesn't show in JEI. I look the tracker, and look like this issue appen before, but, yeah, the issue is back now...

I can find the recipe using the command `/FruzstratedsGuide` and it work, but, in JEI impossible to see the recipe. This can be caused by an update of JEI, but, for sure this have to be fixed.

Version:
Minecraft: 1.12.2
Forge: 14.23.5.2824
EnderTanks: 1.6.8
Just Enough Items: 4.15.0.268

Thank's
Fireztonez

**Comment by ShetiPhian** [comment]
I don't have any special JEI handling and the recipes are regular json defined shaped recipes.
JEI should see them without issues.

Maybe the files are not loading. Are the tanks craftable?

**Comment by Fireztonez** [comment]
Yes the tank is craftable, but we can't see any recipes when I press "R" on the tank...

But I just try with only EnderTank, JEI and ShetiPhianCore and the recipe is visible, so, it look like a Mod conflict, but I really don't know what mod can caused that issue.

The problem in my modpack is fixed, because I change the recipe of the EnderTank. If I find the mod how caused an issue I will let you know, unless, if no body else have that issue, I think the issue can be closed now.
